desktop/libs/aws/src/aws/s3/s3fs.py

```diff
@@ -362,20 +362,19 @@ def rmtree(self, path, skipTrash=True):
       key = self._get_key(path, validate=False)
 
       if key.exists():
-        to_delete = [key]
         dir_keys = []
 
         if self.isdir(path):
-          dir_keys = key.bucket.list(prefix=path)
-          to_delete = itertools.chain(dir_keys, to_delete)
+          _, dir_key_name = s3.parse_uri(path)[:2]
```

Comment:
Seems like `_` is unused here.

Reply:
Yes, we intentionally ignored that return value because the bucket_name we already got from above and this again will become redundant.
cf: https://stackoverflow.com/questions/5893163/what-is-the-purpose-of-the-single-underscore-variable-in-python

Reply:
Why not extract only the second element? 
`dir_key_name = s3.parse_uri(path)[1]`